css/at-rules/media.json

```
@@ -1437,16 +1437,37 @@
             "description": "<code>prefers-reduced-data</code> media feature",
             "support": {
               "chrome": {
-                "version_added": false,
-                "notes": "See <a href='https://crbug.com/1051189'>bug 1051189</a>."
+                "version_added": "85",
+                "notes": "See <a href='https://crbug.com/1051189'>bug 1051189</a>.",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "enable-experimental-web-platform-features",
```

There's some back story here in #6009 (which still needs some PRs to formally fix), but I've made a few suggestions to try to be more consistent with other entries like this

```suggestion
                    "name": "#enable-experimental-web-platform-features",
```